### PR TITLE
Remove php 7.1 from allowed failures

### PR DESCRIPTION
**Pull request by Jurigag**
Hello!

* Type: code quality
* Link to issue:

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)?
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I wrote some tests for this PR.

Small description of change: This removes php 7.1 from allowed failures to test php 7.1

Thanks

### CODE REVIEW

**Comment by sergeyklay** [comment]
Refs: https://github.com/phalcon/cphalcon/issues/12055

**Comment by Jurigag** [comment]
And https://github.com/phalcon/zephir/pull/1415 too since ti's fixing 7.1

**Comment by sergeyklay** [comment]
@Jurigag I will keep this open to testing 7.1. Feel free to commit here in order to restart build

File: phalcon/mvc/view/engine/volt.zep [hunk 1]
```diff
@@ -35,7 +35,7 @@ class Volt extends Engine
 
 	protected _compiler;
 
-	protected _macros;
+	protected _macros = [];
```
Comment (sergeyklay) [inline phalcon/mvc/view/engine/volt.zep]: PHP doesn't support default array values

https://github.com/phalcon/zephir/issues/1255

**Review by sergeyklay — CHANGES_REQUESTED** [review]

File: phalcon/mvc/view/engine/volt.zep (hunk 1 above)
Comment (Jurigag) [inline phalcon/mvc/view/engine/volt.zep]: So i need to set it in construct method perhaps?

**Comment by Jurigag** [comment]
@sergeyklay ready to merge, will you merge ito 3.2.x or i should cherry pick it?

**Comment by sergeyklay** [comment]
Could you cherry pick it please?

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**